flows/definition/testdata/test_migrations.json

```diff
@@ -0,0 +1,372 @@
+[
+    {
+        "legacy_test": {
+            "type": "between",
+            "min": "12",
+            "max": "34"
+        },
+        "expected_case": {
+            "uuid": "������������������������������������",
+            "type": "has_number_between",
+            "arguments": ["12", "34"],
+            "exit_uuid": "c072ecb5-0686-40ea-8ed3-898dc1349783"
+        },
+        "expected_localization": {}
+    },
+    {
+        "legacy_test": {
+            "type": "contains",
+            "test": {
+                "eng": "yes yeah",
+                "fra": "oui"
+            }
+        },
+        "expected_case": {
+            "uuid": "������������������������������������",
+            "type": "has_all_words",
+            "arguments": ["yes yeah"],
+            "exit_uuid": "c072ecb5-0686-40ea-8ed3-898dc1349783"
+        },
+        "expected_localization": {
+            "fra": {
+                "������������������������������������": {
+                    "arguments": ["oui"]
+                }
+            }
+        }
+    },
+    {
+        "legacy_test": {
+            "type": "contains_any",
+            "test": {
+                "eng": "yes yeah",
+                "fra": "oui"
+            }
+        },
+        "expected_case": {
+            "uuid": "������������������������������������",
+            "type": "has_any_word",
+            "arguments": ["yes yeah"],
+            "exit_uuid": "c072ecb5-0686-40ea-8ed3-898dc1349783"
+        },
+        "expected_localization": {
+            "fra": {
+                "������������������������������������": {
+                    "arguments": ["oui"]
+                }
+            }
+        }
+    },
+    {
+        "legacy_test": {
+            "type": "contains_only_phrase",
+            "test": {
+                "eng": "yes yeah",
+                "fra": "oui"
+            }
+        },
+        "expected_case": {
+            "uuid": "������������������������������������",
```

Comment:
Wildcard chars so the tests can match two strings but ignore parts like these UUIDs which are non deterministic. Using `�` instead of `?` means one doesn't have to worry about real question marks.